docs/source/model_design/risk_factors.rst

```diff
@@ -200,6 +200,100 @@ Confounding
 Intermediates
 """""""""""""
 
+An intermediate variable as discussed in this section is defined as a variable 
+between an exposure and outcome in a sequential causal chain, as demonstrated 
+in the diagram below:
+
+.. image:: basic_int_diagram.svg
+
+A (simplified) example that can demonstrate such a causal pathway is the 
+relationship between the exposure of cigarette smoking, which is causally 
+related to the intermediate variable of accumulated tar in lungs, which in 
+turn is causally related to lung cancer (note: for the purposes of this 
+example, assume that there is no direct causal relationship between cigarette 
+smoking and lung cancer other than through the intermediate variable of 
+accumulated tar in lungs).
+
+.. image:: smoking_intermediate_example.svg
+
+Now, let's say that we plan to enact an advertising campaign to reduce smoking 
+in hopes of reducing population lung cancer rates. Our expected causal pathway 
+would look like this:
+
+.. image:: smoking_intervention_example.svg
+
+Therefore, as we increase intervention coverage, we would exect a decrease in 
+population lung cancer rates, as these two variables are located on the same 
+causal pathway. 
+
+However, let's imagine that a brand new hypothetical medication was just 
```

Comment:
You could perhaps give headings to your example sections. This feels like a new type of example, because before, smoking was a proxy for tar in lungs, and now, it's a poor measure of tar in lungs without information on medication coverage.

Reply:
Also-- you had mentioned/asked for suggestions on alternatives examples of intermediates. I'm thinking about categories:

1.  good proxies (maybe smoking/tar in lungs)
2.  worse proxies but still known direct relationship (smoking/tar in lungs, despite existence of medication)
3. assumed or observed proxies?

I'm not sure if in #3 you leave mediation. For example, if you have data on years of education, and you know this is linked to income, and you want to model some outcome for which income is a driver of change---could you treat and model this as mediation, even if perhaps the relationship is quite a bit more complicated?

This seems like it should be disambiguated somewhere, but you suggested during coworking that perhaps that occurs after all of these causal relationship sections.

Reply:
Thanks for the suggestions, Beatrix! I added some sub-headings and agree that they make the section more read-able. So far, I included "An Example of Uncomplicated Intermediates", "An Example of Complicated Intermediates", and "Other Considerations". Let me know if you think these could be improved!

As for your point about potential overlap between issues like intermediates and mediation, I'm thinking that it might be good to include a discussion of this in a section called "Causal Inference and the Simulation Science Team" (similar to the section "Causal Inference in the Global Burden of Disease Study" that was added in PR#201). where we talk about how we might consider how we might be limited by data gaps in applying our understanding of these concepts (defined in their individual sections) to specific interventions or risks and how such data gaps might affect the certainty in our model results. Let me know how this sounds. 

Thanks :)